### ISSUE TITLE
DownloadItem Typings

### ISSUE THREAD

**Issue by mrfunkycold** [electron/electron-typescript-definitions#71]
Documentation for DownloadItem: https://electron.atom.io/docs/api/download-item/

This isn't an issue since it does type correctly. I wanted to propose a change.

Electron Documentation states the following for DownloadItem.
```
Event: ‘updated’
Returns:

event Event
state String
Emitted when the download has been updated and is not done.

The state can be one of following:

progressing - The download is in-progress.
interrupted - The download has interrupted and can be resumed.
```

Which ends up typing as: 
```
on(event: 'updated', listener: (event: Event,
                                    state: string) => void): this;
```

This gives the impression that the state field can be something other than `progressing` or `interrupted`.

I wanted to see if its possible to change the documentation (which will hopefully change the typings to be more concise).

Basically, the typings file to be:
```
on(event: 'updated', listener: (event: Event,
                                    state: 'progressing' | 'interrupted') => void): this;
```

**Comment by MarshallOfSound** [comment]
This is possible, we just have to update the docs 👍 

Basically if you add a comment after the `state` type which says "Can be thing or thing" it will be parsed and added to the typescript docs.

I.e.

```markdown
  * `state` String - Can be `progressing` or `interrupted`.
```

**Comment by mrfunkycold** [comment]
@MarshallOfSound Awesome. Sounds doable. 

I don't mind helping out get this change in, there are other statuses within the downloadItem module I would like to see fixed as well. How does one go about making a change?

**Comment by MarshallOfSound** [comment]
@mrfunkycold Sorry I dropped the ball on this, didn't see your reply 😢 If you're still interested in making this change you just need to update the raw markdown file over at:

https://github.com/electron/electron/blob/master/docs/api/download-item.md#event-updated

With the syntax I indicated above:

```markdown
* `name` Type - Can be `thing 1`, `thing 2` or `thing 3`
```

**Comment by TechnicalSoup** [comment]
I would be happy to do these changes if @mrfunkycold isn't able to get to them.

**Comment by mrfunkycold** [comment]
@MarshallOfSound no worries! sorry for not responding. These messages were all in my spam folder for some reason 😞 

@TechnicalSoup Please feel free to create the PR. I am heads down on something else at the moment. Won't be able to get to it 😢
